src/fixup.js

```diff
@@ -297,6 +297,40 @@
     request.send();
   }
 
+  /* Dark mode toggle */
+  const darkCss = document.querySelector('link[href="dark.css"]');
+  if (darkCss && matchMedia("(prefers-color-scheme)").matches) {
```

Comment:
I think you want to remove the `matchMedia("(prefers-color-scheme)").matches`. If someone doesn't have a preferred default color scheme, seems OK for them to still get a toggle!

Reply:
agree

Reply:
addressed: https://github.com/w3c/tr-design/pull/243/commits/9efe263c396ab35f998f9d313532176e6851367e